Controller/ConsoleController.php

```diff
@@ -79,19 +91,31 @@ public function consoleAction(): Response
                     'working_dir' => getcwd(),
                     'environment' => $this->environment,
                     'commands'    => $this->application->all(),
+                    'sync'        => $request->get('sync', 'false') === 'true',
                 ]
             )
         );
     }
 
+    private function ensureSessionStarted(): void
+    {
+        if ($this->session->isStarted()) {
```

Comment:
Shouldn't it be :
```suggestion
        if (!$this->session->isStarted()) {
```
?